src/client/sandbox/event/active-window-tracker.ts

```diff
@@ -1,31 +1,30 @@
 import SandboxBase from '../base';
+/*eslint-disable no-unused-vars*/
+import MessageSandbox from './message';
+/*eslint-enable no-unused-vars*/
 
 const WINDOW_ACTIVATED_EVENT   = 'hammerhead|event|window-activated';
 const WINDOW_DEACTIVATED_EVENT = 'hammerhead|event|window-deactivated';
 
 export default class ActiveWindowTracker extends SandboxBase {
-    messageSandbox: any;
+    _isIframeWindow: boolean;
+    _activeWindow: Window;
+    _isActive: boolean;
 
-    isIframeWindow: boolean;
-    activeWindow: Window;
-    isActive: boolean;
-
-    constructor (messageSandbox) {
+    constructor (private readonly _messageSandbox: MessageSandbox) { //eslint-disable-line no-unused-vars
         super();
 
-        this.messageSandbox = messageSandbox;
-
-        this.isIframeWindow = null;
-        this.activeWindow   = null;
-        this.isActive       = null;
+        this._isIframeWindow = false;
```

Comment:
Let's use the `private` modifier there and in other places if a class property starts with the underscore symbol.